provider/src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/builder/model/UpgradeGuidanceRiskView.java

```diff
@@ -0,0 +1,87 @@
+package com.synopsys.integration.alert.provider.blackduck.collector.builder.model;
+
+import java.math.BigDecimal;
+
+import com.synopsys.integration.alert.common.rest.model.AlertSerializableModel;
+import com.synopsys.integration.blackduck.api.generated.component.ComponentVersionUpgradeGuidanceLongTermVulnerabilityRiskView;
+import com.synopsys.integration.blackduck.api.generated.component.ComponentVersionUpgradeGuidanceShortTermVulnerabilityRiskView;
+
+public class UpgradeGuidanceRiskView extends AlertSerializableModel {
+    private BigDecimal critical;
+    private BigDecimal high;
+    private BigDecimal medium;
+    private BigDecimal low;
```

Comment:
Is the number of vulnerabilities not discrete? I assume using `BigDecimal` is not a mistake, but it seems so odd.

Reply:
I use BigDecimal here because that is what we get in the response from Black Duck. I can probably change this to be integer

Reply:
No need to change it. It's just odd that BlackDuck's response is not discrete.